chevah/keycert/ssh.py

```diff
@@ -690,6 +697,8 @@ def _fromString_PUBLIC_OPENSSH(cls, data):
         @raises BadKeyError: if the blob type is unknown.
         """
         blob = base64.decodestring(data.split()[1])
+        if cls._guessStringType(blob) != 'blob':
+            raise BadKeyError("Fail to parse OpenSSH key content.")
```

Comment:
We can also refactor this as


```
    return cls._fromString_BLOB(blob, key_name='OpenSSH')
```

and _fromString_BLOB will include the key_name for all the raised messages